client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -46,12 +43,9 @@ const EditorSharingAccordion = React.createClass( {
 		}
 
 		const skipped = PostMetadata.publicizeSkipped( post );
-		const targeted = connections.filter( ( connection ) => {
-			return ! includes( skipped, connection.keyring_connection_ID );
-		} );
-		const targetedServices = serviceConnections.getServicesFromConnections( targeted );
+		const targeted = filter( connections, ( { keyring_connection_ID: ID } ) => ! ( ID in skipped ) );
 
-		return map( targetedServices, 'label' ).join( ', ' );
+		return map( uniqBy( targeted, 'service' ), 'label' ).join( ', ' );
```

Comment:
It's also available without Lodash as [`Array#reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce), which you could use here instead (related discussion at https://github.com/Automattic/wp-calypso/pull/10173#discussion_r93444050).